cpp/include/cudf/strings/detail/convert/fixed_point.cuh

```diff
@@ -121,7 +121,7 @@ __device__ thrust::optional<int32_t> parse_exponent(char const* iter, char const
  * @brief Converts the string in the range [iter, iter_end) into a decimal.
  *
  * @tparam DecimalType The decimal type to be returned
- * @param iter The beginning of the string. Unless iter >= iter_end, iter is dereferenced
+ * @param iter The beginning of the string unless iter >= iter_end, iter is dereferenced
```

Comment:
```suggestion
 * @param iter The beginning of the string
```
This seems awkward and probably not necessary. If it should remain, then the original wording with the included '.' should be kept.